src/utility/setLocalStorage.tsx

```diff
@@ -0,0 +1,30 @@
+import { Settings } from '../settings/settingsConfig';
+
+class LocalStorage {
```

Comment:
The file is called setLocalStorage but the class says LocalStorage. Confusing... hmm what do we set here?